src/cloudant/database.py

```diff
@@ -586,11 +590,8 @@ def missing_revisions(self, doc_id, *revisions):
         url = posixpath.join(self.database_url, '_missing_revs')
         data = {doc_id: list(revisions)}
 
-        resp = self.r_session.post(
-            url,
-            headers={'Content-Type': 'application/json'},
-            data=json.dumps(data)
-        )
+        resp = self.r_session.post(url, headers={'Content-Type': 'application/json'},
+                                   data=json.dumps(data))
```

Comment:
Can you return the statement to its original form?  Again, the coding standard we've adopted.